lib/bibliothecary/parsers/pypi.rb

```diff
@@ -252,18 +268,16 @@ def self.parse_requirements_txt(file_contents, options: {}) # rubocop:disable Li
             deps << result.merge(
               type: type
             )
-          else
-            match = line.delete(" ").match(REQUIREMENTS_REGEXP)
-            next unless match
-
+          elsif (match = line.delete(" ").match(REQUIREMENTS_REGEXP))
             deps << {
               name: match[1],
               requirement: match[-1] || "*",
               type: type,
             }
           end
         end
-        deps
+
+        deps.uniq
```

Comment:
🎱 Depending on how many dupes there are, could be more memory efficient to initialize deps as a Set object and `.to_a` the return here

Reply:
I thinkkk `pipdeptree`'s top-level of deps in the JSON should never have duplicates, but I just included it here to be extra safe 🤷🏻